server/config/admin/admin-pages.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "title": "Home",
+    "ref": "/admin",
+    "view": "/admin/home"
+  },
+  {
+    "title": "Page Editors",
+    "ref": "/admin/editor?page=home",
```

Comment:
Let's not add a query to this string, instead in the router.get(admin/editor) let's render home if no page is given - much cleaner imo